source/js/components/multipage-nav/multipage-nav.jsx

```diff
@@ -18,19 +18,30 @@ export default class MultipageNav extends React.Component {
   }
 
   render() {
+    let activeLinkLabel;
     let links = this.props.links.map((link, index) => {
       let className = `opp-link${ link.isActive ? ` opp-link-active` : `` }`;
```

Comment:
We should probably change all `opp-link` and `opp-link-active` class names to something more general like `multipage-link` and `multipage-link-active`. If this sounds sane let's tackle it in a follow-up ticket.

Reply:
Yeah that makes sense